rep-I0001.rst

```diff
@@ -109,43 +109,46 @@ The dynamic joint state is meant to mimic both the ROS JointState and FollowJoin
         accelerations[] 
         effort[]
         position_desired[]
-        position_errors[]
+        position_error[]
         velocity_desired[]
-        velocity_errors[]
+        velocity_error[]
+        accel_desired[]
+        accel_error[]
         effort_desired[]
         effort_error[]
     
     
 Dynamic Group Status
----------
-The dynamic group status is meant to mimic both the ROS-I RobotStatus message.  See the RobotStatus message[#rbt_stat]_ for field descriptions.::
+--------------------
+The dynamic group status is meant to mimic both the ROS-I RobotStatus message.  See the RobotStatus message [#rbt_stat]_ for field descriptions.::
 
     length: true message/data length 
     header: standard msg_type, comms_type, reply_code fields 
     num_groups: # of motion groups included in this message 
     group[]: # length of this array must match num_groups
         id:   control-group ID for use on-controller 
-        mode:
-        e_stopped:
         drives_powered:
-        motion_possible:
-        in_motion:
+        e_stopped:
+        error_code:
         in_error:
+        in_motion:
+        mode:
+        motion_possible:
```

Comment:
Hm. The linked header and the actual order used in the [RobotStatus::load(..)](https://github.com/ros-industrial/industrial_core/blob/12a74a1f9f26aea0ee075edaf4c84473bd8e112a/simple_message/src/robot_status.cpp#L163-L165) and [RobotStatus::unload(..)](https://github.com/ros-industrial/industrial_core/blob/12a74a1f9f26aea0ee075edaf4c84473bd8e112a/simple_message/src/robot_status.cpp#L185-L187) implementations would seem not to match with each other.